src/control/cmd/dmg/storage_test.go

```diff
@@ -1,330 +0,0 @@
-//
-// (C) Copyright 2019-2020 Intel Corporation.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
-// The Government's rights to use, modify, reproduce, release, perform, display,
-// or disclose this software are subject to the terms of the Apache License as
-// provided in Contract No. 8F-30005.
-// Any reproduction of computer software, computer software documentation, or
-// portions thereof marked with this legend must also reproduce the markings.
-//
-
-package main
-
-import (
-	"context"
-	"strings"
-	"testing"
-
-	"github.com/pkg/errors"
-
-	"github.com/daos-stack/daos/src/control/common"
-	ctlpb "github.com/daos-stack/daos/src/control/common/proto/ctl"
-	"github.com/daos-stack/daos/src/control/lib/control"
-	"github.com/daos-stack/daos/src/control/logging"
-	"github.com/daos-stack/daos/src/control/system"
-)
-
-func TestStorageCommands(t *testing.T) {
-	storageFormatReq := &control.StorageFormatReq{Reformat: true}
-	storageFormatReq.SetHostList([]string{})
```

Comment:
Might have missed something, but is this test duplicated somewhere else? Moved? Seems like we should keep this one at least.

Reply:
whoops, adding back as part of https://github.com/daos-stack/daos/pull/3589 , good catch . thanks